src/main/java/org/dependencytrack/integrations/defectdojo/DefectDojoClient.java

```diff
@@ -50,10 +50,13 @@ public class DefectDojoClient {
     private static final SimpleDateFormat DATE_FORMAT = new SimpleDateFormat("yyyy-MM-dd");
     private final DefectDojoUploader uploader;
     private final URL baseURL;
+    private final boolean verifyFindings;
 
-    public DefectDojoClient(final DefectDojoUploader uploader, final URL baseURL) {
+    public DefectDojoClient(final DefectDojoUploader uploader, final URL baseURL, final boolean verifyFindings) {
```

Comment:
Consider adding the `verifyFindings` flag as a method parameter to `uploadDependencyTrackFindings` and `reimportDependencyTrackFindings` methods instead, just like how it's done for `engagementId`.